src/utils/quick-open.js

```diff
@@ -126,27 +126,43 @@ export function formatSources(sources: SourcesMap): Array<QuickOpenResult> {
     .toJS();
 }
 
-export function groupSharedChars(parts: string[], letters: string[]) {
-  const shared = parts.filter(char => letters.includes(char));
+export function groupFuzzyMatches(input: string, query: string) {
+  const parts = input.toLowerCase().split("");
+  const queryChars = query.toLowerCase().split("");
+  const shared = parts.filter(char => queryChars.includes(char));
   const title = [];
   let matched;
+  let missed;
   parts.forEach((char, i) => {
     if (shared.includes(char)) {
       if (!matched) {
-        matched = [char];
+        matched = { type: "match", value: input[i] };
```

Comment:
It's important that we use `input[i]` for the value portion as the others are all lowercased.